packages/appeals-service-api/src/repositories/appeals-repository.js

```diff
@@ -15,6 +15,19 @@ class AppealsRepository extends MongoRepository {
 		return await this.findOneByQuery({ _id: id });
 	}
 
+	/**
+	 *
+	 * @param {any} email
+	 * @return {Promise<any>}
+	 */
+	async getByEmail(email) {
+		const result = await this.getAllDocumentsThatMatchQuery({
+			'appeal.email': email,
+			'appeal.lpaCode': { $exists: true } // only return if an actual appeal has been started
```

Comment:
is appeal.email not enough here, isn't it always set after lpaCode?

Reply:
yeah on the return flow it doesn't have it

Reply:
I guess my question would be is it possible to avoid creating an appeal in session if they've clicked

Reply:
Oh okay I see what you mean, probably possible, I'll try it